source/slang/parameter-binding.cpp

```diff
@@ -1037,7 +1048,13 @@ static void completeBindingsForParameter(
 
             continue;
         }
-
+        else if (kind == LayoutResourceKind::GenericResource)
+        {
+            bindingInfo.space = (size_t)-1;
```

Comment:
Is there a motivation for using space `-1` instead of just the default `0` that everything else uses?